engines/dfc_provider/app/services/enterprise_builder.rb

```diff
@@ -23,6 +23,11 @@ def self.enterprise(enterprise)
       e.registerSemanticProperty("ofn:long_description") do
         enterprise.long_description
       end
+      e.registerSemanticProperty("ofn:contact_name") do
+        # This could be expressed as dfc-b:hasMainContact Person with name.
+        # But that would require a new endpoint for a single string.
+        enterprise.contact_name
```

Comment:
Do you think we should eventually adhere to the DFC format ?

Reply:
> Do you think we should eventually adhere to the DFC format ?

Maybe. Probably yes. I wanted to get this done though and I didn't want to make it more complicated for Grace to parse the result and store it in Airtable. But if we need this information in other DFC compatible apps then yes, we should add the DFC format.